packages/core/admin/admin/src/content-manager/hooks/useLazyComponents/index.js

```diff
@@ -36,23 +39,19 @@ const useLazyComponents = (componentUids = []) => {
       setStore(Object.fromEntries(componentStore));
     };
 
-    if (componentUids.length && loading) {
+    if (loading) {
       /**
        * These uids are not in the component store therefore we need to get the components
        */
-      const newUids = componentUids.filter((uid) => !componentStore.get(uid));
-
       const componentPromises = newUids.map((uid) => {
         const customField = customFieldsRegistry.get(uid);
 
         return customField.components.Input();
       });
 
-      if (componentPromises.length > 0) {
-        lazyLoadComponents(newUids, componentPromises);
-      }
+      lazyLoadComponents(newUids, componentPromises);
     }
-  }, [componentUids, customFieldsRegistry, loading]);
+  }, [newUids, customFieldsRegistry, loading]);
```

Comment:
Now we're not using `componentUids` in that hook so just changed it to new and correct variable